utils/node_local_test.py

```diff
@@ -1946,12 +1949,12 @@ def _run_test():
             print('Calling {}'.format(fn))
             try:
                 pt.container = create_cont(conf,
-                                           pool.uuid,
+                                           pool.id(),
```

Comment:
There is only one pool created by NLT (with the label NLT) and I'd deliberately left it so that the uuid was used in some instances and the label used in others.  I'm not sure if it's still the case after this PR but it would be good to check and keep that usage somewhere.

There's also currently a bug where pool is given by uuid but container by name so maybe it would be better to write a test specifically for this.

Reply:
I only changed this in the places where either a uuid or label would work, in order to prefer the label when it's present.